requirements.txt

```diff
@@ -35,4 +35,5 @@ Flask-Bootstrap4==4.0.2
 PyQRCode==1.2.1
 Flask-Limiter==1.4
 factory_boy>=3.2.1
+sqlalchemy-utils
```

Comment:
where is this used?

Reply:
It's used in the `conftest.py`. But I have quite likely added it by mistake, I forgot we have a `requirements-test.txt` where this is already added. I'll remove it again.

Reply:
Removed now.